src/json.hpp.re2c

```diff
@@ -618,7 +700,8 @@ class basic_json
         boolean,        ///< boolean value
         number_integer, ///< number value (integer)
         number_float,   ///< number value (floating-point)
-        discarded       ///< discarded by the the parser callback function
+        discarded,      ///< discarded by the the parser callback function
+        number_unsigned ///< number value (unsigned integer)
```

Comment:
Could you please move `number_unsigned` behind `number_integer` (with the required adjustment in the ordering), just to have the numbers grouped together?